src/analysis/souffle/examples/multimic/multimic.authlogic

```diff
@@ -0,0 +1,132 @@
+// UserA is the device owner. UserA has a high level of trust in their own
+// device.
+"UserA" says {
+  "UserA" hasVoiceRecordingPreference("AlwaysOK").
```

Comment:
Is this valid syntax? @aferr

Reply:
Yes: see [this paragraph](https://github.com/google-research/raksha/tree/main/rust/tools/authorization-logic#attributes-and-canactas) (especially the first sentence, but the rest will make it clear why this is different from just `hasVoiceRecordingPreference("UserA", "AlwaysOK")`)